pkg/traceql/enum_attributes.go

```diff
@@ -9,6 +9,7 @@ const (
 	AttributeScopeResource
 	AttributeScopeSpan
 	AttributeScopeUnknown
+	AttributeScopeTrace
```

Comment:
addressed elsewhere but trace attributes don't make sense to me. i feel like all trace level fields are intrinsic

Reply:
:+1: done